meta-resin-common/classes/image_types_resin.bbclass

```diff
@@ -160,7 +160,7 @@ IMAGE_CMD_resin-sdcard () {
 	# After creating the extended partition the next logical parition needs a IMAGE_ROOTFS_ALIGNMENT in front of it
 	START=$(expr ${START} \+ ${IMAGE_ROOTFS_ALIGNMENT})
 	END=$(expr ${START} \+ ${CONFIG_SIZE_ALIGNED})
-    parted -s ${RESIN_SDIMG} unit KiB mkpart logical fat32 ${START} ${END}
+    parted -s ${RESIN_SDIMG} unit KiB mkpart logical fat16 ${START} ${END}
```

Comment:
It is not the wrong tab @telphan . Is the same as the one before. We need to cleanup the file generally and fix tabs but this is not the scope of this change,